__tests__/app.test.js

```diff
@@ -79,4 +80,73 @@ describe('Express app',() => {
 
 
     });
+
+    describe.only('PATCH /api/articles/:article_id',() => {
+        it('200: /api/articles/1 when article_id exists',() => {
+            return request(app).patch('/api/articles/1').query({inc_votes:1}).expect(200);
```

Comment:
Need more assertions! You should check that it actually incremented the votes in the database.